pkg/operator/ceph/controller/spec.go

```diff
@@ -663,6 +667,14 @@ func PrivilegedContext(runAsRoot bool) *v1.SecurityContext {
 
 // LogCollectorContainer runs a cron job to rotate logs
 func LogCollectorContainer(daemonID, ns string, c cephv1.ClusterSpec) *v1.Container {
+	if c.LogCollector.MaxLogSize == "" {
+		c.LogCollector.MaxLogSize = "500M"
+	}
+
+	if c.LogCollector.Periodicity == "" {
```

Comment:
This is not allowed by the schema validation.

Reply:
This

```
Pattern=`^(hourly|daily|weekly|monthly)$`
```

does not allow empty strings.

Reply:
adding `^$|^(hourly|daily|weekly|monthly)$` will work right?

Reply:
Yes but how is the question related to my query?